contracts/JBTiered721Delegate.sol

```diff
@@ -26,13 +26,18 @@ import './libraries/JBIpfsDecoder.sol';
   Ownable: Includes convenience functionality for checking a message sender's permissions before executing certain transactions.
 */
 contract JBTiered721Delegate is IJBTiered721Delegate, JB721Delegate, Votes, Ownable {
+  using Checkpoints for Checkpoints.History;
   //*********************************************************************//
   // --------------------------- custom errors ------------------------- //
   //*********************************************************************//
 
   error NOT_AVAILABLE();
   error OVERSPENDING();
 
+  mapping(address => mapping(uint256 => address)) private _tierDelegation;
+  mapping(address => mapping(uint256 => Checkpoints.History)) private _delegateTierCheckpoints;
+  mapping(uint256 => Checkpoints.History) private _totalTierCheckpoints;
```

Comment:
Should we make the above mappings public so we can extend the functionality from external contracts (ex. tier governance)? I don't see the data being exposed anywhere

Reply:
Yeah. I just took the pattern from the base class. Probably missed the view fns